kong/clustering/data_plane.lua

```diff
@@ -124,12 +125,16 @@ function _M:communicate(premature)
     return
   end
 
+  local labels = dp_labels.decode(kong.configuration.cluster_dp_labels)
```

Comment:
missing error handling

Reply:
if an error happens here it means the same string that was already decoded successfully during validation in the conf_lader now fails decoding, which should never happen, but I guess it won't hurt to add a warning if that happens (which should only be the case if cjson experiences an intermittent failure in decoding the input)